src/xpcc/architecture/platform/driver/gpio/stm32/gpio.hpp.in

```diff
@@ -312,9 +322,15 @@ public:
 		constexpr uint8_t bit_pos = (pin & 0x03) << 2;
 		constexpr uint16_t syscfg_mask = (0x0f) << bit_pos;
 		constexpr uint16_t syscfg_value = (port_nr & (0x0f)) << bit_pos;
+%% if target is stm32f1
```

Comment:
The F1 have their own gpio driver, because they are different enough to warrant that.
This switch is already [implemented here](https://github.com/roboterclubaachen/xpcc/blob/develop/src/xpcc/architecture/platform/driver/gpio/stm32f1/gpio.hpp.in#L233-L235).